pkg/token/token.go

```diff
@@ -564,14 +564,23 @@ func (v tokenVerifier) Verify(token string) (*Identity, error) {
 	req.Header.Set(clusterIDHeader, v.clusterID)
 	req.Header.Set("accept", "application/json")
 
+	stsEndpointType := metrics.InvalidSTSEndpoint
```

Comment:
we already validate the endpoint above, is there a chance for it to be invalid?

Reply:
no. i will remove that